libs/ui/inputs/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -72,6 +72,11 @@ export class AutocompleteComponent
         this.updateInputValue(value.currentValue)
       }
     }
+    if (this.inputRef) {
+      this.inputRef.nativeElement.value = value?.currentValue?.title
+        ? value.currentValue.title
+        : ''
+    }
```

Comment:
to move this section in `updateInputValue` method
```suggestion
    if (this.inputRef) {
      this.inputRef.nativeElement.value = (value as any)?.title || ''
    }
```